src/storage/fileset.proto

```diff
@@ -21,6 +21,14 @@ message DeleteFile {
   string path = 1;
 }
 
+// TODO: Src and dst naming?
```

Comment:
I don't think we need to bikeshed the naming too much, but we definitely need comments, after reading the implementation it's clear that `dst` is prepended to all paths after `src` is trimmed, but we should tell the caller about that here so they don't have to read the code.